fastembed/late_interaction/colbert.py

```diff
@@ -175,8 +175,8 @@ def __init__(
         self._model_dir = self.download_model(
             self.model_description, self.cache_dir, local_files_only=self._local_files_only
         )
-        self.mask_token_id = None
-        self.pad_token_id = None
+        self.mask_token_id = int()
```

Comment:
seems like a complicated way of writing `self.mask_token_id = 0`
not sure whether we'd like to initialise it with 0, i would rather keep None